subprojects/packagefiles/openssl/meson_options.txt

```
@@ -0,0 +1 @@
+option('openssl_asm', type : 'feature', value : 'auto', description : 'Use ASM')
```

```suggestion
option('asm', type : 'feature', value : 'auto', description : 'Enable platform-specific assembly optimizations')
```

(Options usually don't have a package-specific prefix.)